Client/Frontend/Browser/TopTabsViewController.swift

```diff
@@ -107,9 +107,9 @@ class TopTabsViewController: UIViewController {
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
-    
-    override func viewWillAppear(_ animated: Bool) {
-        super.viewWillAppear(animated)
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
         self.reloadData()
```

Comment:
No. Unfortunately, this bug isn't just about calling reloadData before the view is created. It's more about other parts of the app that might add tabs after the TopTabs is created but before its displayed. For example the new tab that is created on first run.